apypie/exceptions.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import print_function, absolute_import
+
+from sys import version_info
+
+
+class Printable(object):
+    message = None
```

Comment:
should `message` be `''` by default? otherwise:

```python
>>> message = None
>>> message.encode('utf-8')
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    message.encode('utf-8')
AttributeError: 'NoneType' object has no attribute 'encode'
```